tests/rptest/tests/delete_records_test.py

```diff
@@ -171,7 +173,7 @@ def test_delete_records_topic_start_delta(self):
     # occurs at the moment a failure is injected.
     @cluster(num_nodes=3, check_for_storage_usage_inconsistencies=False)
     @parametrize(truncate_point="at_segment_boundary")
-    @parametrize(truncate_point="within_segment")
+    @parametrize(truncate_point="random_segment")
```

Comment:
Should be random_offset?

Reply:
ah that is a typo tho for this commit, will update